scripts/build/rollup.config.js

```diff
@@ -0,0 +1,34 @@
+const babel = require('rollup-plugin-babel');
+const commonjs = require('rollup-plugin-commonjs');
+const closure = require('rollup-plugin-closure-compiler-js');
+const replace = require('rollup-plugin-replace');
+
+module.exports = {
+  input: 'src/index.js',
+  output: {
+    file: 'lib/react-dom-lite.js',
+    format: 'cjs',
+  },
+  plugins: [
+    replace({
+      __DEV__: JSON.stringify(process.env.NODE_ENV === 'development'),
+    }),
+    babel(),
+    commonjs(),
+    closure({
+      compilationLevel: 'SIMPLE',
+      languageIn: 'ECMASCRIPT5_STRICT',
+      languageOut: 'ECMASCRIPT5_STRICT',
+      env: 'CUSTOM',
+      applyInputSourceMaps: false,
+      processCommonJsModules: false,
+    }),
+  ],
+  external: [
```

Comment:
should this be `Object.keys(require('pacakge.json).dependencies)` so we don't miss one?

Reply:
Actually yes.

Reply:
My bad. It doesn't cut it. utils like dom-helpers/style have to referred exactly the same as they are required.

Reply:
aww